packages/docs/pages/apps/gitlab/triggers.md

```diff
@@ -0,0 +1,36 @@
+---
+favicon: /favicons/gitlab.svg
```

Comment:
@gh-kdk you specified the SVG file here, but it needs to be placed as `docs/pages/public/favicons/gitlab.svg`